code/modules/resleeving/machines.dm

```diff
@@ -438,6 +433,9 @@
 		manip_rating += M.rating
 	blur_amount = (48 - manip_rating * 8)
 
+	var/total_rating = manip_rating + scan_rating
+	sickness_duration = (45 - (total_rating-4)*1.875) MINUTES		// 45 minutes default, 30 minutes with max non-anomaly upgrades, 15 minutes with max anomaly ones
```

Comment:
Considering a round only lasts 6 hours, it maaayyy be wise to turn this down a smidge. Sorry we keep giving you all this contradictory information. :U

Personally I would be satisfied with even just a ten or fifteen minute cooldown until the player's resleeving sickness is gone. That matches our old respawn allowance rate, and matches our current suggested respawn rate for players even in vore deaths.

Reply:
Note that it starts off at 45 and goes down to 15 when fully upgraded. I would say current numbers are fine, since, any round that will have any serious amount of resleeving will probably have an active enough science to upgrade the machines.

Reply:
30 mins base please. You're going to make deadshifts hell otherwise.

Reply:
My main argument for all the times has been that 15 minutes is, with assumption that you really died remotely somewhere, what it will take you to re-prepare, re-gear, and potentially catch up on events leading up to the death, basically time you're least expected to engage in combat after resleeve regardless of circumstance. 45 minutes maximum is high, but its to make sure that upgrading the resleeving machinery is still highly incentivized, and makes it so that you get less mechanical punishment as a reward mechanical aspect of upgrading. While RP-based deaths (aka vore/sceny deaths) are now only slightly influenced by upgrades and are as a general thing easier to deal with. On deadshifts resleever might not get upgraded, but same argument would go for combat, its very unlikely for there to be severe enough situation that it is gameruining on that lowpop. If expedition does happen, lowpop would struggle with both getting amount of people for it right as well as sometiems even doing the resleeving itself at all and if combat event happens in lowpop, overseeing admin can probably ensure that sickness won't get in the way of things or play around it. 30 minutes is what you would expect resleever to be like on your average populated round, and taking the about 10-15 minutes of re-gearing, re-briefing and preparing, that makes it about 15-20 minutes of _actual_ cooldown, unless you are REALLY rushing to get back into action.

Reply:
"You're going to make deadshifts hell otherwise."

Apparently they missed the part where we removed resleeving sickness from regular vore scenes.